includes/Modules/Tag_Manager/Web_Tag.php

```diff
@@ -86,11 +86,11 @@ private function render_no_js() {
 		$iframe_src = 'https://www.googletagmanager.com/ns.html?id=' . rawurlencode( $this->tag_id );
 
 		?>
-		<!-- Google Tag Manager (noscript) added by Site Kit -->
+		<!-- <?php esc_html_e( 'Google Tag Manager snippet added by Site Kit', 'google-site-kit' ); ?> -->
 		<noscript>
 			<iframe src="<?php echo esc_url( $iframe_src ); ?>" height="0" width="0" style="display:none;visibility:hidden"></iframe>
 		</noscript>
-		<!-- End Google Tag Manager (noscript) -->
+		<!-- <?php esc_html_e( 'End Google Tag Manager snippet added by Site Kit', 'google-site-kit' ); ?> -->
```

Comment:
This part may have been missed in the ACs in that I think we probably want to keep the `noscript` variant in the comment here. @felixarntz ?

Reply:
Good point @aaemnnosttv, yes let's keep that "(noscript)" suffix in here. cc @kuasha420

Reply:
@felixarntz Thanks!